### DISCUSSION TITLE
Creating new `content types`

### DISCUSSION THREAD

**Discussion by joeyfezster** [strapi/strapi#8137] [Legacy Topics]
Hi,

I had strapi project I started with `--quickstart` that I then migrated to use MySQL.
The migration was fine, but now I can't see the option to create new content types (see images below)

I have tested that my database_user has all the permissions necessary for the strapi DB in MySQL, but still no joy.

Expected:
![image](https://user-images.githubusercontent.com/11221303/94856046-6b7e9880-03fd-11eb-9bec-b16ebc02a15d.png)

Problem: no create new options
![image](https://user-images.githubusercontent.com/11221303/94855985-543fab00-03fd-11eb-9479-cee599896a40.png)

**Comment by derrickmehaffy** [comment]
Hi there :)

Just a heads up, we will be getting rid of GitHub Discussions sometime in Q1 2021, in the future can you please ask your question on our [Forum](https://forum.strapi.io/c/support). Your question is fairly easily to answer so I'll go ahead and provide you one now. Thanks!

In this case, it appears your project has been started outside of the `development` environment and isn't using the `strapi develop` script from the package.json, meaning you won't be able to create or update any content types. To read more about why this is, see: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#why-can-t-i-create-or-update-content-types-in-production-staging
